scripts/package.ps1

```diff
@@ -30,12 +30,14 @@ foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\src", "System
 }
 
 Ensure-Nuget-Exists
-Write-Host "** Creating BrotliNative NuGet packages. **"
-$brotliExternalFile = "$repoRoot\external\BrotliNative.nuspec"
-Invoke-Expression "$nugetPath pack $brotliExternalFile -Version 0.0.1 -o $packagesPath"
-
-if (!$?) {
-    Write-Error "Failed to create NuGet package for project $brotliExternalFile"
+Write-Host "** Creating NuGet packages from nuspec. **"
+foreach ($file in [System.IO.Directory]::EnumerateFiles("$repoRoot\external", "*.nuspec", "AllDirectories")) {
+    Write-Host "Creating NuGet package for $file..."
+    Invoke-Expression "$nugetPath pack $file -Version 0.0.1 -o $packagesPath"
```

Comment:
The version here is going to be a problem for multiple nuspec. You should move the version back to the nuspec file.